### ISSUE TITLE
Report if arguments in "Assert.That" are switched

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#2169]
We should report if an `Assert.That` checks a literal or enum member and propose to fix it by switching the arguments.

```c#
Assert.That(StringComparison.Ordinal, Is.EqualTo(xyz.GetComparisonToUse());
```

should be changed into:
```c#
Assert.That(xyz.GetComparisonToUse(), Is.EqualTo(StringComparison.Ordinal));
```

Same is for booleans and other literals (numbers, null, characters, etc.)
